install/install.go

```diff
@@ -1051,6 +1405,68 @@ func K8sUninstaller(c *k8s.Client, o Options) error {
 	return nil
 }
 
+// K8sUninstaller for karmor uninstall
+func K8sUninstaller(c *k8s.Client, o Options) error {
```

Comment:
Since the `kubearmor-operator` helm release is installed first, then KubeArmor. Any reason for listing pods, instead of present helm releases for checking if KubeArmor is installed or not?

It might lead to situations like below:
Operator is in `imagePullBackOff` and no other kubearmor resources exist, unable to uninstall.
![image](https://github.com/kubearmor/kubearmor-client/assets/54525605/8a0a3ff0-a142-43f3-b8fb-fd7a11029397)

Reply:
Great catch! Thank you. I will refactor the checking mechanism.

Reply:
Update: The [if](https://github.com/kubearmor/kubearmor-client/blob/e5913634d9e286d36f536be256169547bb953290/install/install.go#L1412) block is first checking for user supplied namespace correctness and the [else](https://github.com/kubearmor/kubearmor-client/blob/e5913634d9e286d36f536be256169547bb953290/install/install.go#L1419) block is auto-finding the namespace kubearmor is installed in if the user did not specify a namespace. Once namespace is determined [it](https://github.com/kubearmor/kubearmor-client/blob/e5913634d9e286d36f536be256169547bb953290/install/install.go#L1445) checks weather the installation was done through helm and procceds to choose standard or legacy uninstallation.